src/QsCompiler/Core/SymbolTable/NamespaceManager.fs

```diff
@@ -408,16 +408,7 @@ type NamespaceManager(syncRoot: IReaderWriterLock,
 
                 false, errs
             else
-                match GetEntryPoints() |> Seq.tryHead with
-                | None -> isExecutable, errs
-                | Some (epName, epSource) ->
-                    let msgArgs = [ sprintf "%s.%s" epName.Namespace epName.Name; epSource ]
-
-                    errs.Add
-                        (offset,
-                         range |> orDefault |> QsCompilerDiagnostic.Error(ErrorCode.OtherEntryPointExists, msgArgs))
-
-                    false, errs
```

Comment:
We should coordinate this with microsoft/qsharp-runtime#528, since allowing multiple entry points will break C# generation without that PR. Since that PR can't be fully tested without also using this PR, maybe using a feature branch would be better?

Reply:
This PR is dependent on the runtime PR https://github.com/microsoft/qsharp-runtime/pull/528, but that runtime PR is not dependent on this PR. That PR has tests to ensure continuity of existing behavior and just makes room for this PR to work. That PR will need to be merged in first, but I don't think a feature branch is needed here.

Reply:
That would also help make sure we can test the alphas from that feature branch against IQ#'s entry point generator.

Reply:
The runtime PR's purpose is to add support for multiple entry points, but it currently has no tests against multiple entry points. I'm concerned that we'll merge both of these PRs, then add driver tests for multiple entry points and realize that there are bugs/design issues that only came up after using it in practice. That could cause more changes/revisions to the entry point driver code. Using a feature branch to make sure it's thoroughly tested would avoid the potential churn on main.

Reply:
While this PR needs to be coordinated with the runtime PR, I don't think we need to require having a feature branch every time that is the case. Given that both PRs are ready and tested, let's move forward with this.

Reply:
@bettinaheim, I did not mean that a feature branch is needed every time a PR needs to be coordinated across multiple repos. This comment was made when the runtime PR did not have tests for multiple entry points, so I requested a feature branch until the tests were added. Those tests have now been added directly to the runtime PR, so I don't think a feature branch is needed anymore.